source/geometry/FloatBox2D.ooc

```diff
@@ -40,7 +40,7 @@ FloatBox2D: cover {
 	rightCenter ::= FloatPoint2D new(this right, this center y)
 	topCenter ::= FloatPoint2D new(this center x, this top)
 	bottomCenter ::= FloatPoint2D new(this center x, this bottom)
-	empty ::= this size empty
+	empty ::= this size x <= 0.0f || this size y <= 0.0f
```

Comment:
We need to add more functions to Float extension, as this will fail for my favourite float point case when `size x, size y = (0.1 + 0.1 + 0.1) - 0.3`.